config/initializers/assets.rb

```diff
@@ -1,3 +1,5 @@
-if Rails.application.config.should_show_easy_login
-  Rails.application.config.assets.precompile += %w( admin.css public.css easy_sign_in.js )
-end
+Rails.application.config.assets.precompile += %w(
+  admin.css
+  easy_sign_in.js
```

Comment:
I agree with @pzi and @swebb in theory - I did this work because I found it jarring that precompilation was a once off process and the env variables that determined whether or not this file was required could be changed at any point.

If the consensus is to continue using the env variable (albeit correctly) to determine which files to precompile then I will change it to be that way.

I think the only opinions we are missing are @osiro and @lordmortis